### PR TITLE
Refs [TASK][34317] Hanlde get data all fixtures

### PR DESCRIPTION
**Pull request by lanhnv0108**
## Related Tickets
- [#Task 34317: Handle get data all fixtures](https://edu-redmine.sun-asterisk.vn/issues/34317)
## WHAT
- Handle get data
## Evidence (Screenshot or Video)
image

Category | View Point | Description | Expected Reviewer Answer | Self review | Reviewer2 (name)
--- | --- | --- | --- | --- | ---
Conventions | Does the code follow Sun* coding style and coding conventions? | https://github.com/framgia/coding-standards/tree/master/eng/android | YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Redmine | Does the ticket follow Sun* Redmine working process?  | https://github.com/framgia/Training-Guideline/blob/master/WorkingProcess/redmine/redmine.md| YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Documentation | Is there any incomplete code? If so, should it be removed or flagged with a suitable marker like ‘TODO’? |  | YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
## Notes (Optional)

### CODE REVIEW

File: app/src/main/java/com/sun/uefascore/data/source/remote/FixtureRemoteDataSource.kt [hunk 1]
```diff
@@ -23,6 +23,18 @@ class FixtureRemoteDataSource private constructor():
         ).execute(baseUrl)
     }
 
+    override fun getAllFixture(
+        season: String,
+        listener: OnFetchDataJsonListener<MutableList<Fixture>>
+    ) {
+        val baseUrl =
+            Constant.BASE_URL + TypeFootball.FIXTURE.path + Constant.BASE_LEAGUE_ALL + Constant.BASE_SEASON + season
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/sun/uefascore/data/source/remote/FixtureRemoteDataSource.kt]: reformat code

File: app/src/main/java/com/sun/uefascore/screen/fixtures/FixturesFragment.kt [hunk 2]
```diff
@@ -1,5 +1,6 @@
 package com.sun.uefascore.screen.fixtures
 
+import android.annotation.SuppressLint
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/sun/uefascore/screen/fixtures/FixturesFragment.kt]: remove

File: app/src/main/res/layout/fragment_fixtures.xml [hunk 3]
```diff
@@ -16,7 +16,7 @@
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
             android:orientation="vertical"
-            app:layout_constraintGuide_percent="0.072992700729927" />
+            app:layout_constraintGuide_percent="0.07299270072992" />
```
Comment (tuanlvt-1117) [inline app/src/main/res/layout/fragment_fixtures.xml]: kéo thả hay sao ra vị trí thế này chú

Comment (lanhnv0108) [reply app/src/main/res/layout/fragment_fixtures.xml]: em chia tỉ lệ ra số đó anh em không làm trong cho nó chuẩn nảy em xóa số 7 để commit lại đó a

File: app/src/main/java/com/sun/uefascore/data/source/remote/FixtureRemoteDataSource.kt (hunk 1 above)
Comment (lanhnv0108) [reply app/src/main/java/com/sun/uefascore/data/source/remote/FixtureRemoteDataSource.kt]: em reformat nó tự cho xún hàng luôn a

File: app/src/main/res/layout/fragment_fixtures.xml (hunk 3 above)
Comment (tuanlvt-1117) [reply app/src/main/res/layout/fragment_fixtures.xml]: để 0.07 là được rồi e

File: app/src/main/java/com/sun/uefascore/data/source/remote/FixtureRemoteDataSource.kt (hunk 1 above)
Comment (tuanlvt-1117) [reply app/src/main/java/com/sun/uefascore/data/source/remote/FixtureRemoteDataSource.kt]: mỗi dấu + xuống 1 hàng e

Comment (lanhnv0108) [reply app/src/main/java/com/sun/uefascore/data/source/remote/FixtureRemoteDataSource.kt]: Dạ anh

Comment (lanhnv0108) [reply app/src/main/java/com/sun/uefascore/data/source/remote/FixtureRemoteDataSource.kt]: đã fix

**Review by tuanlvt-1117 — APPROVED** [review]

**Pull request merged**